shared/js/background/chrome-events.es6.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * NOTE: this needs to be the first listener that's added
+ *
+ * on FF, we might actually miss the onInstalled event
+ * if we do too much before adding it
+ */
+const ATB = require('./atb.es6')
+
+chrome.runtime.onInstalled.addListener(function(details) {
+    // only run the following section on install and on update
+    if (details.reason.match(/install|update/)) {
+        ATB.updateATBValues();
+    }
+
+    if (details.reason.match(/install/)) {
+        ATB.openPostInstallPage()
```

Comment:
All the post-install page opening used to live in the handler. I've tried to move as much of the actual logic to the modules themselves.

(I've done a similar thing with the `onBeforeRedirect` handle and `tabManager`.)